src/main/wlp/server.xml

```diff
@@ -1,24 +1,24 @@
-<server description="Liberty beta">
+<server description="Liberty JTA Sample">
 
-<featureManager>
-<feature>localConnector-1.0</feature>
-<feature>servlet-3.1</feature>
-<feature>jsonp-1.0</feature>
-<feature>jdbc-4.1</feature>
-<feature>jndi-1.0</feature>
-<feature>cdi-1.2</feature>
-</featureManager>
+    <featureManager>
+        <feature>localConnector-1.0</feature>
+        <feature>servlet-3.1</feature>
+        <feature>jsonp-1.0</feature>
+        <feature>jdbc-4.1</feature>
+        <feature>jndi-1.0</feature>
+        <feature>cdi-1.2</feature>
+    </featureManager>
 
+	<httpEndpoint host="*" httpPort="9080" httpsPort="9443" id="defaultHttpEndpoint"/>
 
-<httpEndpoint host="*" httpPort="9080" httpsPort="9443" id="defaultHttpEndpoint"/>
+	<library id="DerbyLib">
+        <file name="${wlp.user.dir}/shared/resource/derby/derby.jar"/>
```

Comment:
change it back to 

`<file name="${shared.resource.dir}/derby/derby.jar"/>`

See KC for the value of ${shared.resource.dir}:
https://www.ibm.com/support/knowledgecenter/SS7K4U_liberty/com.ibm.websphere.wlp.nd.multiplatform.doc/ae/rwlp_dirs.html